src/helix-ui/styles/components/radios.less

```diff
@@ -0,0 +1,262 @@
+@import (reference) 'mixins';
+@import (reference) 'vars';
+
+// base styles
+// reset?
+// layout geometry
+// pristine/valid
+// invalid
+// disabled
+// modern supports
+
+// BASE STYLES
+hx-radio {
+  background-color: @gray-0;
+  border: 1px solid @gray-500;
+  border-radius: 1rem;
+  color: @gray-500;
+  display: inline-block;
+  height: 1rem;
+  vertical-align: baseline;
+  width: 1rem;
+}
+
+// LAYOUT GEOMETRY
+hx-radio-control {
+  -ms-grid-column-align: start;
+  -ms-grid-columns: auto 1fr;
+  -ms-grid-rows: auto auto;
+  display: -ms-grid;
+  display: grid;
+  grid-template-rows: auto auto;
+  grid-template-columns: auto 1fr;
+  grid-template-areas:
+    'ctrl lbl'
+    '.    help';
+  justify-items: start;
+
+  // facade + text
+  > label {
+    // grid child
+    -ms-grid-column: 2;
+    -ms-grid-row: 1;
+    grid-area: lbl;
+
+    // grid parent
+    -ms-grid-column: auto 1fr;
+    -ms-grid-row: auto;
+    display: -ms-grid;
+    display: grid;
+    grid-template-areas: 'facade txt';
+    grid-template-rows: auto;
+    grid-gap: 0.5rem;
+    margin: 0;
+    padding: 0;
+
+    > hx-radio {
+      -ms-grid-column: 1;
+      -ms-grid-row: 1;
+      grid-area: facade;
+      display: none;
+    }
+  }
+
+  > p {
+    -ms-grid-column: 2;
+    -ms-grid-row: 2;
+    grid-area: help;
+    margin: 0;
+  }
+
+  > input[type="radio"] {
+    -ms-grid-column: 1;
+    -ms-grid-row: 1;
+    grid-area: ctrl;
+    margin: 0;
+
+    border-radius: 1rem;
+    display: inline-block;
+    height: 1rem;
+    margin-right: 0.5rem;
+    width: 1rem;
+
+    &::-ms-check {
+      box-sizing: border-box;
+      display: block;
+    }
+  }
+}
+
+// PRISTINE/VALID STATE
+hx-radio-control {
+  > p {
+    color: @gray-750;
+    font-size: 0.75rem;
+  }
+
+  > input[type="radio"] {
+    color: @gray-500;
+
+    &::-ms-check {
+      background-color: @gray-0;
+      border: 1px solid currentColor;
+      color: inherit;
+    }
+
+    &:focus {
+      box-shadow: @focus-glow;
+
+      ~ label > hx-radio {
+        box-shadow: @focus-glow;
+      }
+    }
+
+    &:hover {
+      color: @cyan-500;
+
+      &::-ms-check {
+        background-color: @cyan-50;
+        color: @cyan-500;
+      }
+    }
+
+    &:checked {
+      color: @cyan-900;
+
+      ~ label > hx-radio {
+        border: 1px solid currentColor;
+        color: @cyan-900;
+        border-radius: 2rem;
+        margin: 0;
+        position: relative;
+
+        &::after {
+          content: "";
```

Comment:
pls alphabetize this block (lines 134-142)

Reply:
block alphabetized